node/src/accountant/mod.rs

```diff
@@ -2121,15 +1584,16 @@ mod tests {
             gwei_amount: 10000,
         };
         let more_money_received_params_arc = Arc::new(Mutex::new(vec![]));
+        let payable_dao = PayableDaoMock::new().non_pending_payables_result(vec![]);
```

Comment:
I'm quite surprised that you need a specialized payable DAO for this kind of test. 
I'm unsuccessfully thinking about any possible call of `non_pending_payables()`.
If it's true that you can disable various scan with using NullScanners then I'd say this is pretty much the place to deploy it.